Leanplum-SDK/Classes/LPInbox.m

```diff
@@ -622,4 +657,8 @@ - (void)removeNewsfeedChangedResponder:(id)responder withSelector:(SEL)selector
     [[LPInbox sharedState]  removeInboxChangedResponder:responder withSelector:selector];
 }
 
+- (void)onForceContentUpdate:(LeanplumNewsfeedSyncedBlock)block {
```

Comment:
Do we need to have this also for newsfeed ?

Reply:
Yeah we should until we have a next major version update and remove newsfeed altogether.

Reply:
hm. I think we shouldn't support new features for deprecated newsfeed.

Reply:
yeah i agree with @Enuviel,  it's already planned for removal.

Reply:
I stand corrected. Will remove it.